src/main/scala/net/snowflake/spark/snowflake/io/StageReader.scala

```diff
@@ -14,7 +14,7 @@ import org.slf4j.{Logger, LoggerFactory}
 
 import scala.util.Random
 
-private[io] object StageReader {
+private[snowflake] object StageReader {
```

Comment:
Why changed to snowflake?

Reply:
It is necessary to make sendEgressUsage() to be called for arrow format.